scripts/changelog.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
Got it.  Yea, I think there are two scripts here.  Starting with bash > shelljs transpiled versions sounds like a good start.  I can see the shelljs release as a shelljs org npm package.  You're clear by me if you'd like to get that out the door.  If you're gonna do it right away, feel free to update this PR as well.

At some point, I'll likely pull out the generate changelog functionality of said release script and publish my own module.  Just so it is easy to generate a changelog for any project.